test_apply_lr.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+from TransportationSplitter import *
+
+class TestApplyLR(unittest.TestCase):

```

Comment:
Not sure if we want to use `pytest` instead of `unittest`?
I believe pytest is more feature-rich with the drawback that it is not included in Python Standard Library.
Didn't we also make the switch to `pytest` in the combobulator lately?

Reply:
that's true, pytest is richer in features; brad and i discussed to stick with unittest for now as we're not really needing those features and we want to limit number of package dependencies